xbmc/filesystem/LibraryDirectory.cpp

```diff
@@ -152,7 +152,7 @@ TiXmlElement *CLibraryDirectory::LoadXML(const std::string &xmlFile)
 
   // check the condition
   std::string condition = XMLUtils::GetAttribute(xml, "visible");
-  if (condition.empty() || g_infoManager.EvaluateBool(condition))
+  if (condition.empty() || CServiceBroker::GetGUI()->GetInfoManager().EvaluateBool(condition))
```

Comment:
this should check for nullptr. ideally filesystem would not call into guilib at all

Reply:
okay, will add the check.